pandas/core/frame.py

```diff
@@ -7029,7 +7039,7 @@ def append(self, other, ignore_index=False, verify_integrity=False, sort=False):
         from pandas.core.reshape.concat import concat
 
         if isinstance(other, (list, tuple)):
-            to_concat = [self] + other
+            to_concat = [self] + list(other)
```

Comment:
```suggestion
            to_concat = [self, *other]
```

although docstring clearly states list, and code as is would raise with tuple, so could just change `if isinstance(other, (list, tuple)):` to `if isinstance(other, list):`